test/data/readOnlyTags.yaml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Wouldn't hurt to include these files in a `readOnlyTags.yml.example` file instead.